dspace/config/local.cfg.EXAMPLE

```diff
@@ -247,3 +247,8 @@ db.schema = public
 
 # Maximum size of a multipart request (i.e. max total size of all files in one request)
 #spring.servlet.multipart.max-request-size = 512MB
+
+# Assetstore configurations have moved to config/modules/assetstore.cfg
+# and config/spring/api/bitstore.xml.
+# Additional storage options (e.g. Amazon S3) are available in `assetstore.cfg`
+# assetstore.dir = ${dspace.dir}/assetstore
```

Comment:
@vins01-4science : You seem to have misunderstood my priori request here.  We now have two `assetstore.dir` commented out sections in `local.cfg.EXAMPLE`.  There's this one, and the one on lines 50-52 above.

I'd recommend moving these lines up to *replace* lines 50-52 in this file. It's confusing to have `assetstore.dir` instructions in two places in the same file.

Reply:
I'm sorry for the mistake, now should be correct.
Thank you.